File: MPI/mpicheck/tests/functions/1.0/MPI_Group_range_excl.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2(10,3)
       INTEGER var_3
       INTEGER var_4
        call mpi_group_range_excl(var_0, var_1, var_2, var_3, var_4)
       call pmpi_group_range_excl(var_0, var_1, var_2, var_3, var_4)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Group_range_excl.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Group var_0;
    int var_1;
    int var_2[2][3];
    MPI_Group *var_3;
    int ret;
    /* calls */
    ret = MPI_Group_range_excl(var_0, var_1, var_2, var_3);
    ret = PMPI_Group_range_excl(var_0, var_1, var_2, var_3);
    return 0;
}
